.env-advanced.example

```
@@ -1,3 +1,6 @@
 # Advanced configuration. Normally, it shouldn't be modified
-ETH_NETWORK=mainnet
+ETH_NETWORK=ropsten
+ALLOW_LEGACY_ENTITIES=true
```

```suggestion
ETH_NETWORK=mainnet
```
We shouldn't allow legacy entities by default, and it would be easier for catalyst owners if the default network was mainnet